dnf5daemon-server/package.cpp

```diff
@@ -196,3 +196,157 @@ dnfdaemon::KeyValueMap package_to_map(
     }
     return dbus_package;
 }
+
+void add_string_list(json_object * json_pkg, const char * cattr, const std::vector<std::string> & vector) {
```

Comment:
I believe that `add_string_list` is only internally used in this `cpp` therefore it should be in anonymous namespace or parked as static. Please consider move the code to a namespace.

Reply:
Done.